ultralist/app.go

```diff
@@ -157,66 +145,46 @@ func (a *App) UnarchiveTodo(input string) {
 }
 
 // EditTodo edits a todo with the given input.
-func (a *App) EditTodo(input string) {
+func (a *App) EditTodo(todoID int, input string) {
 	a.Load()
-	id := a.getID(input)
-	if id == -1 {
-		return
-	}
-	todo := a.TodoList.FindByID(id)
+	todo := a.TodoList.FindByID(todoID)
 	if todo == nil {
-		fmt.Println("No such id.")
+		fmt.Println("No todo with that id.")
 		return
 	}
-	parser := &Parser{}
-
-	if parser.ParseEditTodo(todo, input) {
-		a.save()
-		fmt.Println("Todo updated.")
-	}
-}
 
-// ExpandTodo expands a todo.
```

Comment:
expand is not something I used but came via https://github.com/ultralist/ultralist/issues/26